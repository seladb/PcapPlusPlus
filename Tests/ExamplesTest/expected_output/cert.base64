MIIDljCCAzygAwIBAgIQCo1JeqxMTZQQppEcIu5VXjAKBggqhkjOPQQDAjA7MQswCQYDVQQGEwJVUzEeMBwGA1UEChMVR29vZ2xlIFRydXN0IFNlcnZpY2VzMQwwCgYDVQQDEwNXRTIwHhcNMjUwNzA3MDgzNjAwWhcNMjUwOTI5MDgzNTU5WjAZMRcwFQYDVQQDEw53d3cuZ29vZ2xlLmNvbTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABOIGnDZf7vJemhL8tahkEz14Y+BOhjrcDHHcE4771OksUYpbOOOTz0+JeZbftq5IXC44VmsZU+pKsGmKu89XnEGjggJCMIICPjAOBgNVHQ8BAf8EBAMCB4AwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUlmgvJS6fs3uhO7viZhdh6o9RKHgwHwYDVR0jBBgwFoAUdb7Ed66J9kQ3fc+xaB8dGuvcNFkwWAYIKwYBBQUHAQEETDBKMCEGCCsGAQUFBzABhhVodHRwOi8vby5wa2kuZ29vZy93ZTIwJQYIKwYBBQUHMAKGGWh0dHA6Ly9pLnBraS5nb29nL3dlMi5jcnQwGQYDVR0RBBIwEIIOd3d3Lmdvb2dsZS5jb20wEwYDVR0gBAwwCjAIBgZngQwBAgEwNgYDVR0fBC8wLTAroCmgJ4YlaHR0cDovL2MucGtpLmdvb2cvd2UyL2RUTTMtMGhwV2ZFLmNybDCCAQUGCisGAQQB1nkCBAIEgfYEgfMA8QB2ABLxTjS9U3JMhAYZw48/ehP457Vih4icbTAFhOvlhiY6AAABl+Q+Dw4AAAQDAEcwRQIgLDf8SAmt3/qAexy7/glCG9lnV9DiYTFOQG9EHsEBuEgCIQDS1r7QLH5I1pBmm11vGbSeZj3XprnCuw+lNdOgVVGB4wB3AO08S9boBsKkogBX28sk4jgB31Ev7cSGxXAPIN23Pj/gAAABl+Q+DykAAAQDAEgwRgIhAI2cqrXs4PQzVZfv250w0r7M4qHk6QXROvuvTof12jn2AiEAgmTv8N4+YfX6FlegM+4YYoeR74PCNLEeVwIMsG/ioxswCgYIKoZIzj0EAwIDSAAwRQIhAMQrb3CsZP9FbUROCOqEDsokPkiwOKOugJb1ymST+3UNAiBp8hUoQ/Hm4aJd7CTGM4bno4MibUQG711f+ha8xPAFjA==
