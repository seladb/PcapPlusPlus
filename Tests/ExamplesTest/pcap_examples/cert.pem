-----BEGIN CERTIFICATE-----
MIIDljCCAzygAwIBAgIQCo1JeqxMTZQQppEcIu5VXjAKBggqhkjOPQQDAjA7MQsw
CQYDVQQGEwJVUzEeMBwGA1UEChMVR29vZ2xlIFRydXN0IFNlcnZpY2VzMQwwCgYD
VQQDEwNXRTIwHhcNMjUwNzA3MDgzNjAwWhcNMjUwOTI5MDgzNTU5WjAZMRcwFQYD
VQQDEw53d3cuZ29vZ2xlLmNvbTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABOIG
nDZf7vJemhL8tahkEz14Y+BOhjrcDHHcE4771OksUYpbOOOTz0+JeZbftq5IXC44
VmsZU+pKsGmKu89XnEGjggJCMIICPjAOBgNVHQ8BAf8EBAMCB4AwEwYDVR0lBAww
CgYIKwYBBQUHAwEwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUlmgvJS6fs3uhO7vi
Zhdh6o9RKHgwHwYDVR0jBBgwFoAUdb7Ed66J9kQ3fc+xaB8dGuvcNFkwWAYIKwYB
BQUHAQEETDBKMCEGCCsGAQUFBzABhhVodHRwOi8vby5wa2kuZ29vZy93ZTIwJQYI
KwYBBQUHMAKGGWh0dHA6Ly9pLnBraS5nb29nL3dlMi5jcnQwGQYDVR0RBBIwEIIO
d3d3Lmdvb2dsZS5jb20wEwYDVR0gBAwwCjAIBgZngQwBAgEwNgYDVR0fBC8wLTAr
oCmgJ4YlaHR0cDovL2MucGtpLmdvb2cvd2UyL2RUTTMtMGhwV2ZFLmNybDCCAQUG
CisGAQQB1nkCBAIEgfYEgfMA8QB2ABLxTjS9U3JMhAYZw48/ehP457Vih4icbTAF
hOvlhiY6AAABl+Q+Dw4AAAQDAEcwRQIgLDf8SAmt3/qAexy7/glCG9lnV9DiYTFO
QG9EHsEBuEgCIQDS1r7QLH5I1pBmm11vGbSeZj3XprnCuw+lNdOgVVGB4wB3AO08
S9boBsKkogBX28sk4jgB31Ev7cSGxXAPIN23Pj/gAAABl+Q+DykAAAQDAEgwRgIh
AI2cqrXs4PQzVZfv250w0r7M4qHk6QXROvuvTof12jn2AiEAgmTv8N4+YfX6Fleg
M+4YYoeR74PCNLEeVwIMsG/ioxswCgYIKoZIzj0EAwIDSAAwRQIhAMQrb3CsZP9F
bUROCOqEDsokPkiwOKOugJb1ymST+3UNAiBp8hUoQ/Hm4aJd7CTGM4bno4MibUQG
711f+ha8xPAFjA==
-----END CERTIFICATE-----
