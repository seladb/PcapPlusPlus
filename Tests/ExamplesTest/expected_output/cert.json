{
    "version": 3,
    "serialNumber": "a8:d4:97:aa:c4:c4:d9:41:0a:69:11:c2:2e:e5:55:e",
    "issuer": "C=US, O=Google Trust Services, CN=WE2",
    "validity": {
        "notBefore": "2025-07-07 08:36:00",
        "notAfter": "2025-09-29 08:35:59"
    },
    "subject": "CN=www.google.com",
    "subjectPublicKeyInfo": {
        "subjectPublicKeyAlgorithm": "ECDSA",
        "subjectPublicKey": "04:e2:06:9c:36:5f:ee:f2:5e:9a:12:fc:b5:a8:64:13:3d:78:63:e0:4e:86:3a:dc:0c:71:dc:13:8e:fb:d4:e9:2c:51:8a:5b:38:e3:93:cf:4f:89:79:96:df:b6:ae:48:5c:2e:38:56:6b:19:53:ea:4a:b0:69:8a:bb:cf:57:9c:41"
    },
    "extensions": [
        {
            "type": "KeyUsage",
            "isCritical": true
        },
        {
            "type": "ExtendedKeyUsage",
            "isCritical": false
        },
        {
            "type": "BasicConstraints",
            "isCritical": true
        },
        {
            "type": "SubjectKeyIdentifier",
            "isCritical": false
        },
        {
            "type": "AuthorityKeyIdentifier",
            "isCritical": false
        },
        {
            "type": "AuthorityInfoAccess",
            "isCritical": false
        },
        {
            "type": "SubjectAltName",
            "isCritical": false
        },
        {
            "type": "CertificatePolicies",
            "isCritical": false
        },
        {
            "type": "CRLDistributionPoints",
            "isCritical": false
        },
        {
            "type": "CTPrecertificateSCTs",
            "isCritical": false
        }
    ],
    "signatureAlgorithm": "ECDSAWithSHA256",
    "signature": "30:45:02:21:00:c4:2b:6f:70:ac:64:ff:45:6d:44:4e:08:ea:84:0e:ca:24:3e:48:b0:38:a3:ae:80:96:f5:ca:64:93:fb:75:0d:02:20:69:f2:15:28:43:f1:e6:e1:a2:5d:ec:24:c6:33:86:e7:a3:83:22:6d:44:06:ef:5d:5f:fa:16:bc:c4:f0:05:8c"
}
